_posts/2017-07-14-wildfly-elytron-principal-transformers.adoc

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: 'WildFly Elytron - Principal Transformers, Realm Mappings, and Principal Decoders'
 date: 2017-07-14
-tags: elytron PrincipalDecoder PrincipalTransformer RealmMapper wildfly
+tags: principaldecoder principaltransformer realm mapper
```

Comment:
Maybe we could make this principal-decoder and principal-transformer and realm-mapper. WDYT?